### DISCUSSION TITLE
How i can print with vue pdf embed in vue3 app

### DISCUSSION THREAD

**Discussion by Merite15** [hrynko/vue-pdf-embed#101] [Q&A] [answered]

**Comment by hrynko** [comment, marked as answer]
Hi @Merite15,

Regardless of the version of Vue, you'll a need a [template ref](https://vuejs.org/guide/essentials/template-refs.html) to the `vie-pdf-embed` component that displays the document you want to print. Once you have the `ref`, there is a `print` method (see [README](https://github.com/hrynko/vue-pdf-embed#public-methods)).

**Reply by Merite15** [reply]
It's work thanks for your answer
